src/vivarium_census_prl_synth_pop/constants/data_values.py

```diff
@@ -145,4 +145,13 @@ class Taxes(NamedTuple):
 
 
 # Needed to allow HDFs to be filtered on these columns
-DATA_COLUMNS = ["year", "event_date", "survey_date", "tax_year"]
+DATA_COLUMNS = [
+    # Date columns
+    "year",
+    "event_date",
+    "survey_date",
+    "tax_year",
+    # State columns
+    "state",
+    "mailing_address_state",
```

Comment:
Will we ever need to filter by employer state?

Reply:
Highly unlikely, but we can add it later if we do.